WordPress/Classes/Stores/StatsPeriodStore.swift

```diff
@@ -380,7 +380,7 @@ private extension StatsPeriodStore {
 
         let group = DispatchGroup()
 
-        if FeatureFlag.statsNewAppearance.enabled {
+        if AppConfiguration.isJetpack {
```

Comment:
I'm not sure `AppConfiguration.isJetpack` since WordPress simply won't have access to the stats screen. This comment applies to the following commit: https://github.com/wordpress-mobile/WordPress-iOS/pull/21499/commits/961d60d9e17676685f9200b5ea39292606921af9.